File: Test/Test/Cat.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Test
{
    class Cat
    {
        public Cat(string catName, int catAge)
        {
            name = catName;
            age = catAge;
        }

        public string name;
        public int age;

        public void Meow()
        {
            Console.WriteLine(name + " meows");
        }
    }
}

File: Test++/Test++/Cat.cpp
#include "Cat.h"
#include <stdlib.h>
#include <iostream>
#include <string>

using namespace std;

Cat::Cat()
{
	age = 0;
	name = "";
}

Cat::Cat(string catName, int catAge, string catColor)
{
	name = catName;
	age = catAge;
	color = catColor;
}

void Cat::MakeSound()
{
	cout << name << " meows!" << endl;
}
